javascript/react-controls/src/controls/ColumnSpread.tsx

```diff
@@ -23,15 +23,12 @@ export const ColumnSpread: React.FC<ColumnSpreadProps> = memo(
 	function ColumnSpread({ column, onChange, onDelete }) {
 		const handleColumnChange = useCallback(
 			(_e: any, opt: string | undefined) => {
-				onChange && opt && onChange(opt)
+				opt && onChange?.(opt)
 			},
 			[onChange],
 		)
 
-		const handleDeleteClick = useCallback(
-			() => onDelete && onDelete(),
-			[onDelete],
-		)
+		const handleDeleteClick = useCallback(() => onDelete?.(), [onDelete])
```

Comment:
this could be omitted entirely since it is optional on the IconButton anyway

Reply:
We still have to like...call the delete handler right?